js/src/contracts/registry.js

```diff
@@ -101,4 +101,17 @@ export default class Registry {
       return address;
     });
   }
+
+  lookupMeta (_name, key) {
```

Comment:
Wouldn't it make sense to now use this `lookupMeta` method in `lookupAddress` instead of having twice the same code?